sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/security/CertificateNamesGenerator.java

```diff
@@ -0,0 +1,50 @@
+package com.mesosphere.sdk.offer.evaluate.security;
+
+import org.bouncycastle.asn1.x500.X500Name;
+import org.bouncycastle.asn1.x509.GeneralName;
+import org.bouncycastle.asn1.x509.GeneralNames;
+
+/**
+ * A {@link CertificateNamesGenerator} creates relevant names for given service pod.
+ *
+ * // TODO(mh): What about IP per container feature??
```

Comment:
The problem with that is that you can't know the IP before the container actually runs, right? I'd say that as a first pass, we should support DNS alone and make it very clear in documentation. An issue to revisit this should then be added to the backlog.